Bottom: 25e79538538cdb24ff7213aef6168ba7a003a93b
Top:    4a40db1203c52db1d78c2ba0131a9f1b63ad6fa9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 17:24:54 +0100

Refresh of layer-fixes

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 202010c..c8c47db 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1583,7 +1583,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
       }
     }
   } else {
-    // Draw in numerical order when in 3D view
+    // Draw in group order when in 3D view
     for (ngroups = 0, i = 0; i < max_group; i++) {
       int group = reverse_layers ? max_group - 1 - i : i;
